src/App.tsx

```diff
@@ -589,64 +575,53 @@ function App() {
       {messagesDisplays.map((messageDisplay) => {
         return (
           <div
+            className="text-align-center"
             key={getRetryableIdOrDepositHash(messageDisplay)}
             ref={resultRef}
           >
             {
               <>
                 <h3>
-                  Your transaction status on {messageDisplay.l2Network.name}
+                  L2{" "}
+                  {messageDisplay.l2TxHash !== "null" ? (
+                    <ExternalLink
+                      href={`${messageDisplay.explorerUrl}/tx/${messageDisplay.l2TxHash}`}
+                    >
+                      Transaction
+                    </ExternalLink>
+                  ) : (
+                    "Transaction"
+                  )}{" "}
+                  status on {messageDisplay.l2Network.name}
```

Comment:
to my eyes it feels like it could be clear that this line is connected to the line below (i.e., the thing below is the status). Maybe put the two of them in one line and/or add a colon to the end of line 595? (also maybe just disregard)

Reply:
yea i agree... not sure how to make it look better :/
I don't think it needs a colon
but i also feel that if the text is "Retryable ticket" (like how it was), it's not super clear

do you have any idea on how to edit the copy?